services/etl/dags/etl_waqi_readings.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+
+import pendulum
+from airflow.decorators import dag, task
+from api_interface import get_waqi_data as gwd
+from db.db_engine import get_db
+from db.models import Readings_WAQI_Temp
+from util.read_sql import read_sql
+
+
+@dag(
+    schedule=timedelta(minutes=1),
+    start_date=pendulum.datetime(2022, 1, 1, tz="UTC"),
+    catchup=False,
+    tags=["readings"],
+)
+def etl_readings_waqi():
+    """
+    This dag retrieves air quality readings data from World Air Quality Index project: https://aqicn.org/api/
+    """
+        
+    @task()
+    def create_temp_waqi():
+        sql_stmts = read_sql('dags/sql/create_table_readings_waqi_temp.sql')
+        for stmt in sql_stmts:
+            with get_db() as db:
+                result = db.execute(stmt)
+                db.commit()
+
+    @task()
+    def get_readings_waqi():
+        waqi_data = gwd.get_waqi_data()
+        return waqi_data
+
+    @task()
+    def load_readings_waqi_temp(waqi_data):
+       new_row = Readings_WAQI_Temp(**waqi_data)
+       with get_db() as db:
+            result = db.add(new_row)
+            db.commit()
+            db.refresh(new_row)
```

Comment:
ooo nice and fresh

what does this do? lol

Reply:
will research more!